requirements-test.txt

```diff
@@ -0,0 +1,2 @@
+coverage
+coveralls
```

Comment:
move these to `setup.py` too, as it's done here: https://github.com/openwisp/openwisp-utils/blob/master/setup.py#L43-L45, use `test` instead of `qa`